frontend/angular.json

```diff
@@ -24,9 +24,14 @@
               "src/assets"
             ],
             "styles": [
+              "node_modules/@clr/icons/clr-icons.min.css",
```

Comment:
@mikh86 Nice! I've been wanting to check this one out.

Reply:
This UI kit looked super clean, albeit Material UI is awesome, I'm a huge fan of this one for its flat look and simple color palette. Their design patterns look fantastic and remind me of Ant.design.